util/src/main/java/org/killbill/billing/util/config/definition/JaxrsConfig.java

```diff
@@ -49,4 +49,9 @@
     @DefaultNull
     @Description("Base host address to use for redirect URLs")
     String getJaxrsLocationHost();
+
+    @Config("org.killbill.jaxrs.get.ro")
```

Comment:
@reshmabidikar Juste a note to remember to update our config docs with this new property.